sda-commons-server-prometheus/src/main/java/org/sdase/commons/server/prometheus/PrometheusBundle.java

```diff
@@ -91,13 +95,22 @@ private void createPrometheusRegistry(Environment environment) {
 
     Metrics.addRegistry(meterRegistry);
 
+    // JVM and System Metrics
```

Comment:
binding JVM metrics to micrometer and exposing micrometer metrics through prometheus are logically separated one step is defining what goes in the other how it comes out.

my suggestion would be to put them in different methods